### ISSUE TITLE
ERROR: name 'logging' is not defined.

### ISSUE THREAD

**Issue by i-mine** [scylladb/scylla#4922]
*Installation details*
Scylla version (or git commit hash): scylla-3.0.*
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS on Huawei Cloud

when I execute `scylla_setup` or `systemctl start scylla-server`,there is a problem:
```
Aug 29 13:13:27 cassandra scylla_prepare: ERROR: name 'logging' is not defined. Your system can't be tuned until the issue is fixed.
Aug 29 13:13:27 cassandra scylla_prepare: Traceback (most recent call last):
Aug 29 13:13:27 cassandra scylla_prepare: File "/usr/lib/scylla/scylla_prepare", line 69, in <module>
Aug 29 13:13:27 cassandra scylla_prepare: run("{} --options-file /etc/scylla.d/perftune.yaml".format(perftune_base_command()))
Aug 29 13:13:27 cassandra scylla_prepare: File "/usr/lib/scylla/scylla_util.py", line 270, in run
Aug 29 13:13:27 cassandra scylla_prepare: return subprocess.check_call(cmd, shell=shell, stdout=stdout, stderr=stderr)
Aug 29 13:13:27 cassandra scylla_prepare: File "/usr/lib64/python3.6/subprocess.py", line 311, in check_call
Aug 29 13:13:27 cassandra scylla_prepare: raise CalledProcessError(retcode, cmd)
Aug 29 13:13:27 cassandra scylla_prepare: subprocess.CalledProcessError: Command '['/usr/lib/scylla/perftune.py', '--tune', 'disks', '--dir', '/var/lib/scylla/data', '--dir', '/var/lib/scylla/commitlog', '--options-file', '/etc/scylla.d/perftune.yaml']' returned non-zero exit status 1.
Aug 29 13:13:27 cassandra systemd: scylla-server.service: control process exited, code=exited status=1
```

I try to add import logging in /usr/lib/scylla/perftune.py,that works,but I don't want to add this line every time

**Comment by amoskong** [comment]
Commit https://github.com/scylladb/scylla-seastar/commit/f541231a3011420e26fb33e71355415334dcb861 add a logging.warning() for seastar/scripts/perftune.py

After imported logging manually, did you see the warning? 
`Unexpected exception while attempting to access AWS meta server: ....`

**Comment by i-mine** [comment]
yes,I can see the warning,I use the huawei instance, and when I try more times, it will be passed,but not  
always.

**Comment by amoskong** [comment]
@i-mine  what't the warning content?

/cc @tarzanek

**Comment by amoskong** [comment]
Hi @vladzcloudius, I see it's your code (seastar/scripts/perftune.py: __check_host_type()).
We need a fix for 3.1, and check if the unexpected exception is normal.

**Comment by i-mine** [comment]
If I don't import logging ,the scylla_setup will report error as blows:
```
Do you want to enable Network Interface Card (NIC) and disk(s) optimization?
Yes - optimize the NIC queue and disks settings. Selecting Yes greatly improves performance. No - skip this step.
[YES/no]yes
ERROR: name 'logging' is not defined. Your system can't be tuned until the issue is fixed.
Traceback (most recent call last):
  File "/usr/lib/scylla/scylla_sysconfig_setup", line 75, in <module>
    rps_cpus = out('{} --tune net --nic {} --get-cpu-mask'.format(perftune_base_command(), ifname))
  File "/usr/lib/scylla/scylla_util.py", line 280, in out
    return subprocess.check_output(cmd, shell=shell).strip().decode('utf-8')
  File "/usr/lib64/python3.6/subprocess.py", line 356, in check_output
    **kwargs).stdout
  File "/usr/lib64/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/usr/lib/scylla/perftune.py', '--tune', 'disks', '--dir', '/var/lib/scylla/data', '--dir', '/var/lib/scylla/commitlog', '--tune', 'net', '--nic', 'eth0', '--get-cpu-mask']' returned non-zero exit status 1.
NIC queue setup failed. Press any key to continue...
```
@amoskong the warning content is 
```
[YES/no]yes
Do you want to enable Network Interface Card (NIC) and disk(s) optimization?
Yes - optimize the NIC queue and disks settings. Selecting Yes greatly improves performance. No - skip this step.
[YES/no]yes
WARNING:root:Unexpected exception while attempting to access AWS meta server: <class 'socket.timeout'>
```

**Comment by amoskong** [comment]
> If I don't import logging ,the scylla_setup will report error as blows:
> 
> ```
> Do you want to enable Network Interface Card (NIC) and disk(s) optimization?
> Yes - optimize the NIC queue and disks settings. Selecting Yes greatly improves performance. No - skip this step.
> [YES/no]yes
> ERROR: name 'logging' is not defined. Your system can't be tuned until the issue is fixed.
> Traceback (most recent call last):
>   File "/usr/lib/scylla/scylla_sysconfig_setup", line 75, in <module>
>     rps_cpus = out('{} --tune net --nic {} --get-cpu-mask'.format(perftune_base_command(), ifname))
>   File "/usr/lib/scylla/scylla_util.py", line 280, in out
>     return subprocess.check_output(cmd, shell=shell).strip().decode('utf-8')
>   File "/usr/lib64/python3.6/subprocess.py", line 356, in check_output
>     **kwargs).stdout
>   File "/usr/lib64/python3.6/subprocess.py", line 438, in run
>     output=stdout, stderr=stderr)
> subprocess.CalledProcessError: Command '['/usr/lib/scylla/perftune.py', '--tune', 'disks', '--dir', '/var/lib/scylla/data', '--dir', '/var/lib/scylla/commitlog', '--tune', 'net', '--nic', 'eth0', '--get-cpu-mask']' returned non-zero exit status 1.
> NIC queue setup failed. Press any key to continue...
> ```


> @amoskong the warning content is
> 
> ```
> [YES/no]yes
> Do you want to enable Network Interface Card (NIC) and disk(s) optimization?
> Yes - optimize the NIC queue and disks settings. Selecting Yes greatly improves performance. No - skip this step.
> [YES/no]yes
> WARNING:root:Unexpected exception while attempting to access AWS meta server: <class 'socket.timeout'>
> ```

Thanks.   /CC @vladzcloudius

**Comment by avikivity** [comment]
Will be fixed by  42433a2.

**Comment by slivne** [comment]
merged 42433a25a868554c7b309da80c697b43bb815da8

closing
